tools/convert.py

```diff
@@ -67,6 +69,9 @@ def handle_tensors(args, writer, state_dict):
     # - do something about this being awful and hacky
 
     max_name_len = max([len(s) for s in state_dict.keys()]) + 4
+    if (max_name_len - 4) >= 64:

```

Comment:
I think the max name len was just there to make the print look nice lol. I think you could readd this further down like this since it's cleaner and actually prints the key that failed the check:

https://github.com/city96/ComfyUI-GGUF/blob/165212919e1c682f6ff740f9b69d968634b6d89f/tools/convert.py#L147